src/core/namespace.c

```diff
@@ -1697,6 +1772,24 @@ static int mount_overlay(const MountEntry *m) {
         return 1;
 }
 
+static int mount_bpffs(const MountEntry *m) {
+        assert(m);
+
+        _cleanup_close_ int mnt_fd = fsmount(m->bpffs_fd, 0, 0);
+        int r;
+
+        safe_close(m->bpffs_fd);
+
+        if (mnt_fd < 0)
+                log_debug_errno(errno, "Failed to fsmount: %m");
+
+        r = move_mount(mnt_fd, "", AT_FDCWD, mount_entry_path(m), MOVE_MOUNT_F_EMPTY_PATH);
+        if (r < 0)
+                return log_debug_errno(errno, "Failed to move_mount: %m");
```

Comment:
`"Failed to move bpffs mount to %s: %m", mount_entry_path(m)` ?

Reply:
or maybe `m->path_const` which is the destination path without the prefix?